src/ui/layout/mct-tree.vue

```diff
@@ -60,68 +59,45 @@
                     <span class="c-tree__item__view-control c-nav__down"></span>
                 </div>
             </div>
-
-            <tree-item
-                v-for="(ancestor, index) in focusedAncestors"
-                :key="ancestor.id"
-                :node="ancestor"
-                :show-up="index !== 0 && index < focusedAncestors.length && initialLoad"
-                :show-down="false"
-                :left-offset="index * 10 + 'px'"
-                @resetTree="beginNavigationRequest('handleReset', ancestor)"
-            />
-            <!-- loading -->
-            <div
-                v-if="isLoading"
-                :style="indicatorLeftOffset"
-                class="c-tree__item c-tree-and-search__loading loading"
-            >
-                <span class="c-tree__item__label">Loading...</span>
-            </div>
-            <!-- end loading -->
         </div>
 
-        <!-- currently viewed children -->
-        <transition
-            name="children"
-            appear
-            @after-enter="autoScroll"
+        <div
+            ref="scrollable"
+            class="c-tree__scrollable"
+            :style="scrollableStyles()"
```

Comment:
this probably should be a computed property not a method

Reply:
Cleaned this up a bit and was able to remove itemTop computed variable as well.